async_fuse/src/protocol.rs

```diff
@@ -1117,3 +1117,127 @@ struct FuseCopyFileRangeIn {
     pub len: u64,
     pub flags: u64,
 }
+
+/// FUSE ABI types.
+/// It is safe to transmute a `&[u8]` to `&T` when `T: FuseAbiData`.
+pub(crate) unsafe trait FuseAbiData: Sized {}
+
+macro_rules! unsafe_impl_FuseAbiData_for{
```

Comment:
name macro in lowercase, like `unsafe_imple_fuse_abi_data_for`

Reply:
done